packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -208,7 +208,8 @@ export async function load_node({
 					const proxy = new Proxy(response, {
 						get(response, key, _receiver) {
 							async function text() {
-								const body = await response.text();
+								const clonedResponse = response.clone();
+								const body = await clonedResponse.text();
```

Comment:
```suggestion
								const cloned_response = response.clone();
								const body = await cloned_response.text();
```

These should be snake-case for internal variables.

Reply:
Fixed, Thanks! 😊